src/EditorFeatures/CSharpTest/Organizing/OrganizeUsingsTests.cs

```diff
@@ -12,28 +14,31 @@ namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Organizing
 {
     public class OrganizeUsingsTests
     {
-        protected async Task CheckAsync(string initial, string final, bool specialCaseSystem, CSharpParseOptions options = null)
+        protected async Task CheckAsync(string initial, string final, bool placeSystemNamespaceFirst = false, CSharpParseOptions options = null)
```

Comment:
I can make it required.  Just saw a lot of tests passing in random true/false values even when it didn't affect the test.  false is the default in VS.  It seemed reasonable that only tests that were explicitly testing this behavior should change it.  should parse options be non-optional as well?